CryptoLib/CryptoLib/Ldap/OpenLdap.m

```diff
@@ -46,24 +55,47 @@ - (LdapResponse*)search:(NSString*)identityCode {
         }
     }
     
-    LdapResponse *response = [[LdapResponse alloc] init];
+    NSMutableArray *response = [[NSMutableArray alloc] init];
```

Comment:
There is simpler way to create simple object allocation and initialization in Objective-C:
`NSMutableArray *response = [NSMutableArray new];`